cvat-ui/src/components/annotation-page/canvas/views/canvas2d/image-filters.tsx

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2023 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React, { useCallback, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Row, Col } from 'antd/lib/grid';
+import { CombinedState } from 'reducers';
+import Checkbox, { CheckboxChangeEvent } from 'antd/lib/checkbox';
+import Text from 'antd/lib/typography/Text';
+import Slider from 'antd/lib/slider';
+
+import {
+    addImageFilter,
+    removeImageFilter,
+    setupImageFilter,
+} from 'actions/settings-actions';
+import GammaCorrection from 'utils/fabric-wrapper/gamma-correciton';
+import { ImageFilterAlias, filterActive } from 'utils/image-processing';
+
+import './image-setups.scss';
+
+export default function ImageFilters(): JSX.Element {
```

Comment:
I think it is only GammaImageFilter, not all the filters (speaking about the component name)
Otherwise I would propose to move here Brightness, Contrast and Saturation controls, together with Reset button

Reply:
Lets keep it as GammaFilter, I dont really want to make ImageSetupsContent component bigger